pkg/plugins/mellanox/mellanox_plugin.go

```diff
@@ -149,17 +146,16 @@ func configFW() error {
 			glog.Errorf("mellanox-plugin configFW(): failed : %v : %s", err, out)
 			return err
 		}
+		glog.Infof("mellanox-plugin configFW(): %s", out)
 	}
 	return nil
 }
 
-func mstconfigReadData(pciAddress string, attributes ...string) (string, error) {
-	glog.Infof("mellanox-plugin mstconfigReadData(): try to read %s for device %s", attributes, pciAddress)
-
+func mstConfigReadData(pciAddress string) (string, error) {
+	glog.Infof("mellanox-plugin mstConfigReadData(): device %s", pciAddress)
```

Comment:
debug print?

Reply:
you can use an increased verbosity level for these logs (same goes to the rest of my logging comments)

see:
https://godoc.org/github.com/golang/glog#V

Reply:
fixed